data/_ui/tooltips.txt

```diff
@@ -1191,6 +1191,9 @@ tip "no bays?"
 tip "insufficient energy to fire?"
 	`This ship has insufficient energy storage to fire an installed weapon.`
 
+tip "under 20s of battery?"
+	`This ship has less than 20 seconds of combat battery power.`
```

Comment:
Is this leakage from another pr? It's not mentioned in the description

Reply:
It was the reason I did the refactoring in the first place. It probably went in as part of the same commit before I created this branch, but it can be taken out of the PR.

Reply:
Will it be taken out?

Reply:
I took it out — It should be a separate commit and separate PR